hydra-node/json-schemas/api.yaml

```diff
@@ -141,6 +141,22 @@ channels:
           type: request
           method: POST
           bindingVersion: '0.1.0'
+  /protocol-parameters:
+    description: Endpoint to obtain protocol parameters used in hydra-node.
+    servers:
+      - localhost-http
+    subscribe:
+      summary: Get cardano protocol parameters.
+      operationId: getProtocolParameters
+      description: |
+        Clients can request from hydra-node it's protocol parameters used on start.
```

Comment:
Feels duplicate to the upper description and likely reads odd when looking at the rendered api reference. 

Could: only put one description